validators/faq.adoc

```diff
@@ -0,0 +1,155 @@
+= Troubleshooting
+:icons: font
+:toc: macro
+
+ifdef::env-github[]
+
+:tip-caption: :bulb:
+:note-caption: :information_source:
+:important-caption: :heavy_exclamation_mark:
+:caution-caption: :fire:
+:warning-caption: :warning:
+
+toc::[]
+
+endif::[]
+
+
+
+Need help solving an issue? Check to see if this has already been answered below. If you can't find an answer to your issue, reach out to the SKALE Network developer community on discord.  
+
+http://skale.chat/[Ask a question]
+
+== Validator Node
+
+=== How do I change my Node's IP address?
```

Comment:
I think this title and FAQ overall is a bit confusing. Cause in reality you COULDN'T change node IP address. What we're describing here it's basically a new node registration, just with keeping the old SGX key.

Reply:
done